scripts/url-registration.ts

```diff
@@ -0,0 +1,182 @@
+/*****
+ License
+ --------------
+ Copyright © 2020 Mojaloop Foundation
+ The Mojaloop files are made available by the Mojaloop Foundation under the Apache License, Version 2.0 (the 'License') and you may not use these files except in compliance with the License. You may obtain a copy of the License at
+ http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, the Mojaloop files are distributed on an 'AS IS' BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ Contributors
+ --------------
+ This is the official list of the Mojaloop project contributors for this file.
+ Names of the original copyright holders (individuals or organizations)
+ should be listed with a '*' in the first column. People who have
+ contributed from an organization can be listed under the organization
+ that actually holds the copyright for their contributions (see the
+ Mojaloop Foundation organization for an example). Those individuals should have
+ their names indented and be marked with a '-'. Email address can be added
+ optionally within square brackets <email>.
+ * Mojaloop Foundation
+ - Name Surname <name.surname@mojaloop.io>
+
+ * Google
+ - Steven Wijaya <stevenwjy@google.com>
+ --------------
+ ******/
+
+import { program } from 'commander'
+import axios from 'axios'
+
+interface RegistrationValue {
+  targetServiceId: string
+  type: string
+  endpointPath: string
+}
+
+const registrationValues: Array<RegistrationValue> = [
+  // PUT /participants
+  {
+    targetServiceId: 'central-ledger',
+    type: 'FSIOP_CALLBACK_URL_PARTICIPANT_PUT',
```

Comment:
I think we should use the `@mojaloop/central-services-shared` enums here

Reply:
Ohh okay, will update this later

Reply:
Uhmm... sorry I think I might have some misunderstanding here... do you mean making the request to `central-services-shared`? Or use the enum defined there? Because I think the current enum definition is already based on the one defined in `@mojaloop/central-services-shared`

Reply:
I mean instead of hardcoding the strings, import the enums from central-services-shared and use those

Reply:
Ohhh I see... okay, got it! will update later 👍

Reply:
done :-)